flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+    "only-arches": [
+        "x86_64"
```

Comment:
Why? Surely the whole point of building from source is to enable other architectures? Electron supports all our architectures.

Reply:
I have only tested 64-bit build and I don't know whether my `build.sh` can build under 32-bit due to the native modules

Reply:
hmmm. does the yarn generator do per arch dependencies for that sort of thing?

As for everything else - it's pretty easy for us to check on the builders - either with this or a test PR

Reply:
32-bit support added